Convert PHP extension list to actual lists

Crell opened the pull request:
For readability.  I also added some further clarifying notes around which extensions are available on which versions.  I tried to make it clearer, but the list is not consistent enough to really break down any nicer.

pjcdawkins commented on reference/toolstacks/php/configure-php.md:
```diff
@@ -34,25 +34,80 @@ runtime:
-pecl-http (added in 5.6), pgsql, pinba, propro (added in 5.6), pspell,
-raphf (added in 5.6), readline, recode, redis, snmp, sqlite3, ssh2, tidy,
-xcache, xdebug, xhprof, xmlrpc, xsl, and zendopcache (5.4 only)*.
+The following extensions are enabled by default:
```
You have copied and reformatted the list perfectly, but I think it's incomplete

Crell replied:
Is there some way to automate this list?  I considered scripting it, but my Python-fu wasn't quite strong enough yet. :smile:  (And doing it in PHP seemed like a bad one-off.)  It would likely be easier if we're OK with maintaining a separate list for each container version, since it's the weird overlaps that make it difficult to do.

Crell commented:
I added json, and converted the master list to a table derived off of running containers.  Unfortunately extracting what extensions are enabled by default but are not part of base PHP itself is surprisingly challenging, so we'll go with this list for now as it covers the biggies.

pjcdawkins commented:
The table loses a cell at the end 😞 

<img width="385" alt="screen shot 2016-06-22 at 19 57 57" src="https://cloud.githubusercontent.com/assets/1465106/16279614/abc2e550-38b3-11e6-9a40-a0e724cac6ae.png">

Crell commented:
Yeesh. That looks like an HTMLBook bug. The source is clean.  Since it would be an empty cell anyway, do we just go with it, or try to find some work around?  (I'm tempted to just go with it.)
